source/tests/system/nidcpower_driver_api_tests.cpp

```diff
@@ -447,6 +460,38 @@ TEST_F(NiDCPowerDriverApiTest, ConfigureOutputFunctionAndCurrentLevel_Configures
   EXPECT_EQ(expected_output_function_value, actual_output_function_value);
 }
 
+TEST_F(NiDCPowerDriverApiTest, FetchMultiple_ReturnsResultOfExpectedSize)
+{
+  const char* channel_name = "0";
+  const int timeout = 5;
+  const int expected_count = 20;
+  ::grpc::ClientContext context;
+  dcpower::FetchMultipleRequest request;
+  request.mutable_vi()->set_id(GetSessionId());
+  request.set_channel_name(channel_name);
+  request.set_timeout(timeout);
+  request.set_count(expected_count);
+  dcpower::FetchMultipleResponse response;
+
+  // Attribute 'NIDCPOWER_ATTRIBUTE_MEASURE_WHEN' must be set before calling FetchMultiple
```

Comment:
Extra lines should match given when then sections.

Reply:
Fixed.